scene/gui/line_edit.cpp

```diff
@@ -141,7 +141,7 @@ void LineEdit::_backspace(bool p_word, bool p_all_to_left) {
 
 	if (p_all_to_left) {
 		deselect();
-		text = text.substr(0, caret_column);
+		text = text.left(caret_column);
```

Comment:
`caret_column` is a property that is never negative, if it's negative it's already a bug.